pkg/get/tools.go

```diff
@@ -2571,5 +2571,29 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 							{{.Name}}-{{.VersionNumber}}-{{$os}}-{{$arch}}.tar.gz`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "oven-sh",
+			Repo:        "bun",
+			Name:        "bun",
+			Description: "Bun is an incredibly fast JavaScript runtime, bundler, transpiler and package manager – all in one.",
+			BinaryTemplate: `
+							{{$os := ""}}
+							{{ if HasPrefix .OS "ming" -}}
+							{{$os = "windows"}}
+							{{- else if eq .OS "linux" -}}
+							{{$os = "linux"}}
+							{{- else if eq .OS "darwin" -}}
+							{{$os = "darwin"}}
+							{{- end -}}
```

Comment:
Correct me if I'm wrong here, but `bun` doesn't support windows, why should we handle it?

Also all the conditions returns the same as the `.OS` variable, why can't we just use it like `{{ $os := .OS }}` to reduce unnecessary complexity?
Or even better, can't we just use it like `{{ .OS }}` in the final value?